vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -89,6 +89,31 @@ public void customRepresentationIsEdited() {
         Assert.assertEquals("Yes", cell.$("span").first().getText());
     }
 
+    @Test
+    public void customComboBoxIsEdited() {
```

Comment:
The test name doesn't imply anything about the case which we actually want to test: circular reference in the data. With this test description, it would be totally fine to refactor the tests by removing the circular references, and it would no longer serve its original purpose.
Maybe something like:
`customComboBox_circularReferencesInData_isEdited`

Reply:
Updated the name